torch/csrc/inductor/static_cuda_launcher.cpp

```diff
@@ -311,6 +311,10 @@ PyObject* launch_kernel(PyObject* self, PyObject* args) {
           &stream)) {
     return nullptr;
   }
+  if (gridX * gridY * gridZ <= 0) {
```

Comment:
Found this bug from running unit tests with StaticCudaLauncher: triton ignores all calls to kernels when outside of grid boundaries